### DISCUSSION TITLE
Enable activity log in UsersService.resetPassword

### DISCUSSION THREAD

**Discussion by ybelenko** [directus/directus#11857] [Feature Requests]
https://github.com/directus/directus/blob/0bad2744b39ddb4ccbbee867855dc5a9d812d4c8/api/src/services/users.ts#L358-L366

UsersService initiated without accountability there, so activity log is disabled.
I was able to turn it on with:
```diff
diff --git a/api/src/services/users.ts b/api/src/services/users.ts
index d05554bc1..5131ca4d5 100644
--- a/api/src/services/users.ts
+++ b/api/src/services/users.ts
@@ -363,6 +363,11 @@ export class UsersService extends ItemsService {
                const service = new UsersService({
                        knex: this.knex,
                        schema: this.schema,
+                       accountability: {
+                               ...this.accountability,
+                               user: user.id,
+                               admin: true,
+                       },
                });
 
                await service.updateOne(user.id, { password, status: 'active' });
```

With this activity log we can say when user changed his password last time(search in revisions also necessary). There are security policies which requires to reset user password each month/quarter/year.

**Comment by rijkvanzanten** [comment]
Wanna open a PR for this? 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
